src/status_im/transport/core.cljs

```diff
@@ -47,13 +45,8 @@
             (message/resend-contact-messages [])))
 
 (fx/defn stop-whisper
-  "Stops whisper protocol by removing all existing shh filters
-  It is necessary to remove the filters because status-go there isn't currently a logout feature in status-go
-  to clean-up after logout. When logging out of account A and logging in account B, account B would receive
-  account A messages without this."
-  [{:keys [db] :as cofx} callback]
-  (let [{:transport/keys [filters]} db]
-    (fx/merge
-     cofx
-     (transport.filters/stop-filters callback)
-     (publisher/stop-fx))))
+  "Stops whisper protocol"
+  [cofx]
+  (fx/merge
```

Comment:
Not sure `fx/merge` is needed here. Could be just `(publisher/stop-fx cofx)` call.